### PR TITLE
Upgrade deno_std to 0.3.4 and update tests.

### LINKED ISSUE #1: Support Deno 0.3.x
Hi there. Thanks for creating this framework. :) 

I tried to run the example :

```typescript
import pogo from 'https://deno.land/x/pogo/main.js';

const app = pogo.server({ port : 3000 });

app.route({
    method : 'GET',
    path   : '/hello',
    handler() {
        return 'Hello, world!';
    }
});

app.start();
```

But I ran into this error.

```bash
> $ deno server.ts                                                                                                                             [±master ●]
Compiling file:///Users/nilsnh/deno-testing/server.ts
Downloading https://deno.land/x/pogo/main.js
Downloading https://raw.githubusercontent.com/sholladay/pogo/master/dependencies.js
Downloading https://raw.githubusercontent.com/sholladay/pogo/master/lib/respond.js
Downloading https://raw.githubusercontent.com/sholladay/pogo/master/lib/toolkit.js
Downloading https://deno.land/x/http@61a3911be7e01273e13bf35a3a16285f413f0b70/http_status.ts
Downloading https://deno.land/x/http@61a3911be7e01273e13bf35a3a16285f413f0b70/mod.ts
Compiling https://raw.githubusercontent.com/denoland/deno_std/61a3911be7e01273e13bf35a3a16285f413f0b70/http/mod.ts
Downloading https://raw.githubusercontent.com/denoland/deno_std/61a3911be7e01273e13bf35a3a16285f413f0b70/http/http.ts
Downloading https://raw.githubusercontent.com/denoland/deno_std/61a3911be7e01273e13bf35a3a16285f413f0b70/http/deno
Uncaught Other: Import 'https://raw.githubusercontent.com/denoland/deno_std/61a3911be7e01273e13bf35a3a16285f413f0b70/http/deno' failed: 404 Not Found
    at DenoError (js/errors.ts:22:5)
    at maybeError (js/errors.ts:33:12)
    at maybeThrowError (js/errors.ts:39:15)
    at sendSync (js/dispatch.ts:86:5)
    at fetchModuleMetaData (js/os.ts:73:19)
    at _resolveModule (js/compiler.ts:255:38)
    at resolveModuleNames (js/compiler.ts:486:35)
    at compilerHost.resolveModuleNames (third_party/node_modules/typescript/lib/typescript.js:121106:138)
    at resolveModuleNamesWorker (third_party/node_modules/typescript/lib/typescript.js:88311:127)
    at resolveModuleNamesReusingOldState (third_party/node_modules/typescript/lib/typescript.js:88553:24)
```

My versions. I'm on MacOS Mojave 10.14.4.
```bash
> $ deno --version                                                                                                                                  [±master ●]
deno: 0.3.7
v8: 7.4.238
typescript: 3.4.1
```

### PR DESCRIPTION
**Pull request by nilsnh**
I ran the tests and they now pass. I have Prettier installed in my VSCode and it changed single quotes to double quotes I see. 

This should fix #1 .

### CODE REVIEW

**Comment by nilsnh** [comment]
Woops, hmm. It works locally. 🤔

File: dependencies.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
-export * as http from 'https://deno.land/x/http@61a3911be7e01273e13bf35a3a16285f413f0b70/mod.ts';
-export { Status as status } from 'https://deno.land/x/http@61a3911be7e01273e13bf35a3a16285f413f0b70/http_status.ts';
-export { STATUS_TEXT as statusText } from 'https://deno.land/x/http@61a3911be7e01273e13bf35a3a16285f413f0b70/http_status.ts';
+export * as http from "https://deno.land/x/http@61af419bbc5717c2e2552050aacb20ef1b17480b/server.ts";
```
Comment (sholladay) [inline dependencies.js]: I guess this is the line that fixes the README example, right? Any breaking changes I should be aware of?

**Comment by sholladay** [comment]
I fixed the formatting and tested using the README example, but it crashed for me.

I took the README example, modified the import of `main.js` to point to my local repo on this branch, and then put that in a file named `server.js`. When I run `deno server.js`, it crashes with:

```js
/Users/sholladay/Library/Caches/deno/deps/https/raw.githubusercontent.com/denoland/deno_std/v0.3.4/textproto/mod.ts:85:22
                throw new ProtocolError(`malformed MIME header line: ${str(kv)}`);
                      ^
Uncaught ProtocolError: malformed MIME header line: �   ��39/5
    at ProtocolError (file:///Users/sholladay/Library/Caches/deno/deps/https/raw.githubusercontent.com/denoland/deno_std/v0.3.4/textproto/mod.ts:20:5)
    at readMIMEHeader (file:///Users/sholladay/Library/Caches/deno/deps/https/raw.githubusercontent.com/denoland/deno_std/v0.3.4/textproto/mod.ts:95:15)
```

**Comment by nilsnh** [comment]
Hi again, thanks for checking out the PR. I pulled down your updates and ran `deno --reload test.js`, and they pass. You might be on an older version of deno?

image

**Comment by sholladay** [comment]
The tests pass. However, the README example does not. That's what I described. :)

File: dependencies.js (hunk 1 above)
Comment (nilsnh) [reply dependencies.js]: `deno_std` doesn't have a changelog. But [deno's changelog](https://github.com/denoland/deno/releases) for 0.3.4 mentions that the deno module is gone and replaced by a Deno global var. Other than that it's hard to say.

**Comment by nilsnh** [comment]
Aha! Hmm. I got the README example running now by doing `import pogo from "./pogo/main.js";` I have the code in this pull request in a sub folder of the example.

So, the README example might be failing because it fetches code from the current master branch?

**Comment by nilsnh** [comment]
Ah, 

> I fixed the formatting and tested using the README example, but it crashed for me.
> 
> I took the README example, modified the import of `main.js` to point to my local repo on this branch, and then put that in a file named `server.js`. When I run `deno server.js`, it crashes with:
> 
> ```js
> /Users/sholladay/Library/Caches/deno/deps/https/raw.githubusercontent.com/denoland/deno_std/v0.3.4/textproto/mod.ts:85:22
>                 throw new ProtocolError(`malformed MIME header line: ${str(kv)}`);
>                       ^
> Uncaught ProtocolError: malformed MIME header line: �   ��39/5
>     at ProtocolError (file:///Users/sholladay/Library/Caches/deno/deps/https/raw.githubusercontent.com/denoland/deno_std/v0.3.4/textproto/mod.ts:20:5)
>     at readMIMEHeader (file:///Users/sholladay/Library/Caches/deno/deps/https/raw.githubusercontent.com/denoland/deno_std/v0.3.4/textproto/mod.ts:95:15)
> ```

Hmm. Did you also try `deno --reload server.js`, could it be that Deno used some old cached dependencies?

**Comment by sholladay** [comment]
I ended up tracking that error down to a problem with my machine. My browser was attempting to upgrade all requests to HTTPS on localhost due to the fact that I had used an app on localhost with [HSTS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security) enabled. Deno doesn't support HTTPS last I checked and apparently that is the weird error it's spewing out at the moment. Using a different browser or clearing the HSTS cache are two easy solutions.

So, in short, not a problem with our code. :)

Thanks for the patch!
